woocommerce/woocommerce
# Add/cart link in checkout header 44837

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Adds a "Cart Link" block to the distraction free header template. The block only links to the cart (there is no mini cart functionality) and the user can configure the icon and style via inspector:

![Screenshot 2024-10-07 at 12 26 17](https://github.com/user-attachments/assets/384b6d55-f9d7-4810-a8b7-ea4f58096a9d)

This also adds the distraction free template as one of the header options so it can be toggled alongside other options:

![Screenshot 2024-10-07 at 12 29 08](https://github.com/user-attachments/assets/e204eb83-ed15-4023-96a1-0d39545b0c5d)

Closes #44837



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Open appearance editor
2. Go to Templates > Page: Checkout. Try changing icon styles and colors of the cart link block. If the block is missing, reset your templates to default prior to testing.
3. Go to Patterns > Header > Checkout header. Confirm you can switch the design in the inspector and the distraction free template with cart link is one of the options.
4. Save all changes and checkout of the frontend. Confirm the link correctly takes you back to the cart page and respects your settings.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
